tally/statgram.go

```diff
@@ -28,6 +29,7 @@ type Sample struct {
 	valueType   SampleType
 	sampleRate  float64
 	stringValue string
+	replace     bool
```

Comment:
This guy kinda bums me out.  Is there a way to make the booleans more obvious elsewhere? It's really hard for me to know when seeing a structure instantiated what `false` being the last parameter means.